fars/booking/views.py

```diff
@@ -173,7 +173,7 @@ def delete(self, request, booking_id):
         is_admin = _is_admin(request.user, booking.bookable)
         if _is_admin or self.context['unbookable']:
             now = datetime.now(booking.start.tzinfo)
-            removal_level = int(request.GET.get('repeat'))
+            removal_level = int(request.GET.get('repeat')) if request.GET.get('repeat') else 0
```

Comment:
Couldn't this be rewritten as `int(request.GET.get('repeat') or 0)`?

Reply:
How pythonic :unamused: